TTS/trainer.py

```diff
@@ -819,9 +830,12 @@ def test_run(self) -> None:
     def _fit(self) -> None:
         """🏃 train -> evaluate -> test for the number of epochs."""
         if self.restore_step != 0 or self.args.best_path:
-            print(" > Restoring best loss from " f"{os.path.basename(self.args.best_path)} ...")
-            self.best_loss = load_fsspec(self.args.restore_path, map_location="cpu")["model_loss"]
-            print(f" > Starting with loaded last best loss {self.best_loss}.")
+            model_state = load_fsspec(self.args.restore_path, map_location="cpu")
+            # ignore the restoring of the best loss if do not exist the key "model_loss" in the model state
+            if "model_loss" in model_state.keys():
+                print(" > Restoring best loss from " f"{os.path.basename(self.args.best_path)} ...")
+                self.best_loss = model_state["model_loss"]
+                print(f" > Starting with loaded last best loss {self.best_loss}.")
```

Comment:
This is to allow the loading and continuation of training even if the checkpoint does not have the ["model_loss"] key.